hangman/src/index.js

```diff
@@ -0,0 +1,288 @@
+
+// Create main container div
+const mainContainer = document.createElement('div');
+mainContainer.classList.add('main-container');
+
+// Create gallows container
+const gallowsContainer = document.createElement('div');
+gallowsContainer.classList.add('gallows-container');
+gallowsContainer.innerHTML = `
+      <svg id="gallows-svg" width="500" height="581" viewBox="0 0 500 581" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <rect x="211.337" y="34.6662" width="39" height="199.598" transform="rotate(45 211.337 34.6662)" fill="black" stroke="#FFFEFE" stroke-width="3"/>
+        <rect x="69.5" y="1.5" width="39" height="578" rx="3.5" fill="black" stroke="#FFFEFE" stroke-width="3"/>
+        <rect x="386.5" y="34.5" width="39" height="350" rx="3.5" transform="rotate(90 386.5 34.5)" fill="black" stroke="#FFFEFE" stroke-width="3"/>
+        <rect x="498.5" y="540.5" width="39" height="497" rx="3.5" transform="rotate(90 498.5 540.5)" fill="black" stroke="#FFFEFE" stroke-width="3"/>
+        <rect x="333" y="75" width="10" height="74" fill="black"/>
+      </svg>
+    `;
+
+// Create title
+const title = document.createElement('h1');
+title.classList.add('title');
+title.textContent = 'HANGMAN GAME';
+
+// Append gallows and title to the main container part
+const mainContainerPart1 = document.createElement('div');
+mainContainerPart1.classList.add('main-container__part');
+mainContainerPart1.appendChild(gallowsContainer);
+mainContainerPart1.appendChild(title);
+
+// Create game div
+const game = document.createElement('div');
+game.id = 'game';
+
+// Create word to guess paragraph
+const wordToGuess = document.createElement('p');
+wordToGuess.id = 'wordToGuess';
+wordToGuess.innerHTML = '<span id="word"></span>';
+
+// Create hint paragraph
+const hint = document.createElement('p');
+hint.id = 'wordToGuess';
+hint.innerHTML = '<span id="hint"></span>';
+
+// Create guesses paragraph
+const guesses = document.createElement('p');
+guesses.id = 'guesses';
+guesses.innerHTML = '<span id="wrongGuesses"></span>';
+
+// Create status paragraph
+const statusDisplay = document.createElement('p');
+statusDisplay.id = 'status';
+
+// Create keyboard div
+const keyboard = document.createElement('div');
+keyboard.id = 'keyboard';
+
+// Append game elements to the game div
+game.appendChild(wordToGuess);
+game.appendChild(hint);
+game.appendChild(guesses);
+game.appendChild(statusDisplay);
+game.appendChild(keyboard);
+
+// Append game to the main container part
+const mainContainerPart2 = document.createElement('div');
+mainContainerPart2.classList.add('main-container__part');
+mainContainerPart2.appendChild(game);
+
+// Append both parts to the main container
+mainContainer.appendChild(mainContainerPart1);
+mainContainer.appendChild(mainContainerPart2);
+
+// Append the main container to the body
+document.body.appendChild(mainContainer);
+
+const gallows = document.querySelector('.gallows-container');
```

Comment:
You already have this element in variable `gallowsContainer` (See line 14). Why do you need to search again?